DashAI/front/db.json

```diff
@@ -0,0 +1,194 @@
+{
```

Comment:
Wouldn't it be good to call this file dummyData or testData or testDB ?

Reply:
:ok_hand: